kubeai/install.sh

```diff
@@ -5,18 +5,41 @@
 
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-if [ -n "$HF_TOKEN" ]; then
-    echo "Using HF_TOKEN env var"
-elif [ -n "$1" ] && [ -f "$1" ]; then
-    echo "Using given HF token file"
-    HF_TOKEN=$(cat $1)
-else
-    echo "Usage: [HF_TOKEN=<token>] ${0##*/} [hf-token file]"
-    echo "ERROR: HF token missing!"
+error_exit () {
+	name=${0##*/}
+	cat <<EOF
+Usage: [HF_TOKEN=<token>] $name [HF token file] [Prometheus release name]
+
+HuggingFace token for accessing models can be given either as an
+environment variable, or as a name of a file containing the token
+value.
+
+If Prometheus Helm release name is given, Prometheus monitoring is
+enabled for the inference engine instances, and vLLM dashboard
+(configMap) is installed for Grafana.
+
+ERROR: $1!
+EOF
     exit 1
+}
+
+metrics=""
+for arg in "$@"; do
+	if [ -f "$arg" ]; then
```

Comment:
nit: the arg parsing looks somewhat shaky'n'shady but I guess that's ok for this kind of hack/helper script

Reply:
I would think it very unlikely that user's Prometheus release is named exactly the same as some file in the `kubeai/` directory...

PS. I'm wondering about the benefit of the script, when more things are needed to configure. I think it would be clearer if user would just invoke Helm directly (with command copy-pasted from README), in this case with additional `-f monitoring.yaml` argument.